lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -0,0 +1,82 @@
+# The Blockchain Configuration defines the different settings
+# of the blockchain that cannot be changed once the blockchain
+# is started.
+blockchain_configuration:
+  peer_2_peer: true
```

Comment:
I guess in this stage we won't have `peer_2_peer` enabled, but only standalone node

I am still not sure in which state rust node is atm but according to conversation with Nicolas https://github.com/input-output-hk/cardano-wallet/pull/305#issuecomment-495110512 I would assume we should either:
 * set `peer_2_peer` to false and run as a standalone node
 * set `peer_2_peer` to true and connect to the network - but according to what Nicolas said this requires some node configuration to be changed

Also - don't take my word for it as I am just starting to use the node

Reply:
Hm